worlds/seaofthieves/Locations/Feared/CannonsFired.py

```diff
@@ -0,0 +1,138 @@
+import typing
+
+from ..Locations import LocDetails, WebLocation, WebLocationCollection, WebItemJsonIdentifier
+from ..LocationsBase import LocationsBase
+from ...Regions.RegionCollection import RegionNameCollection
+from ...Regions.RegionDetails import Regions
+from ...Items.ItemReqEvalAnd import ItemReqEvalAnd
+from ...Items.ItemReqEvalOr import ItemReqEvalOr
+from ...Locations.LocationSettingOption import LocationSettingOption
+from ...Locations.ScreenData import ScreenData
+
+
+class SettingsCannonsFired:
+    class Balls(LocationSettingOption):
+        pass
+
+    class CursedBalls(LocationSettingOption):
+        pass
+
+    class PhantomBalls(LocationSettingOption):
+        pass
```

Comment:
Did you forget to fill these in or is this intentional?

Reply:
This is intentional. Pretty much these are Enum classes that all happen to have the same settings. In the __init_ of this file you can see how this is used for example. The reason they have nothing in them is because I did not need any extra settings for any of these options. But in the event I did want a custom setting, the property would get added into one of these classes.